lib/libutils/ext/include/compiler.h

```diff
@@ -40,6 +40,9 @@
 #define __section(x)	__attribute__((section(x)))
 #define __data		__section(".data")
 #define __bss		__section(".bss")
+#ifdef __clang__
+#define __SECTION_FLAGS_RODATA
+#else
 /*
  * Override sections flags/type generated by the C compiler to make sure they
```

Comment:
Agreed. The question is: how do you positively identify GCC? I think you can't because Clang defines the GCC macros for compatibility (`__GNUC__` etc.), and it defines `__clang__` in addition.

Reply:
How inconvenient, let's keep it as it is then.